docs/site/tutorials/model_training_walkthrough.ipynb

```diff
@@ -7,7 +7,7 @@
     "id": "QyCcF45zBQ3E"
    },
    "source": [
-    "##### Copyright 2018 The TensorFlow Authors. [Licensed under the Apache License, Version 2.0](#scrollTo=y_UVSRtBBsJk)."
+    "##### Copyright 2020 The TensorFlow Authors. [Licensed under the Apache License, Version 2.0](#scrollTo=y_UVSRtBBsJk)."
```

Comment:
By convention, the copyright line for files in [tensorflow/tensorflow](https://github.com/tensorflow/tensorflow) shows only the original creation year. I think showing the last modified year is nice too, though.

How about changing the copyright year to `Copyright 201x - 2020` for all `.ipynb` files in `docs/site/tutorials`? Other tutorials can be updated later.